docs/userguide/icu4c/packaging.md

```diff
@@ -40,7 +40,7 @@ distribution from conflicting with any libraries that you need. On operating sys
 that do not have a standard C++ ABI (name mangling) for compilers, it is recommended to
 do this special packaging anyway. More details on customizing ICU are available in the
 [User Guide](../).
-The [ICU Source Code Organization](./index/#icu-source-code-organization) section of
+The [ICU Source Code Organization](./index.md#icu-source-code-organization) section of
```

Comment:
The fix of removing the slash is good, but please don't add the .md. The User Guide URLs are nicer/spiffier without the .md. --> https://unicode-org.github.io/icu/userguide/icu4c/index#icu-source-code-organization